### DISCUSSION TITLE
Sign up problem

### DISCUSSION THREAD

**Discussion by avesatan666** [0perationPrivacy/VoIP#47] [Q&A]
I followed the steps from the inteltechniques.com and got everything set up, but now I have a problem with sign up. The button simply doesn't react. 
Any advice on what I might have done wrong and how to fix it?

**Comment by bioscopic** [comment]
Can you be more specific? What button are you referring to?

**Reply by OyArCEST** [reply]
I am having the same issue. when it says after the Heroku steps,  

"You can now launch your new web app within any browser by navigating to the URL mentioned above (https://xxx.herokuapp.com). If necessary, refresh the browser until you see the login page. Next, you must configure each Telnyx or Twilio number you wish to use within this app."

So i navigated to my unique heroku url and Clicked on "Sign Up", Entered a unique username and secure password as instructed the Clicked on "Sign Up" this page is unresponsive and nothing happens. 

Any help please!

**Reply by OyArCEST** [reply]
Issue Fixed: 

Missed this step the MongoDB setup section:

Click "Add IP Address"

therefore the App wasnt connecting to the DB hence no response on the sign up page for the heroku app

**Reply by LazyG** [reply]
I had similar symptoms and the add IP Address resolved as well

**Comment by pantena** [comment]
> I followed the steps from the inteltechniques.com and got everything set up, but now I have a problem with sign up. The button simply doesn't react. Any advice on what I might have done wrong and how to fix it?

Same problem here. "Sign up" button on my unique heroku url is unresponsive. Nothing happens after entering username, password and pressing "Sign up". Now the question is - was something missed while setting things up (instructions on inteltechiques.com differ a bit from what i encountered, but i feel i followed them pretty well) or can it be that any add-ons are interfering (like Ublock, privacy badger etc.)? 

I did manage to sign up and log into online service (https://voip.operationprivacy.com/voip/) just to see if i can. 

Any possible solutions to this? 

p.s i have an added IP adress in MongoDB, so it's not the cause in my case.

**Reply by pantena** [reply]
Ok, a bit embarassing, but now i know what the problem was. Add-ons it is! Before writing above questions i had disabled ublock, but after posting i disabled all add-ons and voila. Was able to sign up, log in and receive a message. So, all good.

**Reply by pantena** [reply]
Here we go again. Today again i cannot log in. Even with all my add-ons disabled. 
So, i went and deleted my current app on heroku and deleted project on mongodb. Started from scratch - created new project, new app, connected with github. And for the life of me cannot sign up in the https://xxxxx.herokuapp.com/voip/signup. My add-ons are disabled. I even tried to sign up from different pc with default browser without any add-ons and "sign up" button doesn't respond. I do have IP address added in mongodb. So, i don't understand what i'm doing wrong.

Can anybody share any thoughts or point in the right direction?

**Reply by pantena** [reply]
Ok, so in this instance it was to do with IP address. When configuring mongodb profile i didn't see option "Allow Access From Anywhere". But i did add a few IP addresses in "IP access list". And even though i was using one of those IP addresses, i couldn't Sing up to https://xxxxx.herokuapp.com/voip/signup.

https://www.carlosroso.com/how-to-allow-access-from-anywhere-in-mongo-atlas/. This did the trick. I removed my previous IP addresses from "IP access list" and replaced them with 0.0.0.0/0. After that was able to sign up and log in.

**Reply by 0perationPrivacy** [reply]
sorry I just saw this. There's a slight confusion here about IP address. Mongodb is asking "which IP address is allowed to access me", the answer is your Heroku instance, NOT your machine IP. Since you don't know your Heroku IP and it may change, you simply allow all (not the tightest security setting, but good enough). and allow all translates to **0.0.0.0/0**
I will update the instructions to include this IP.
Thanks!

**Reply by 0perationPrivacy** [reply]
I believe when you set up the mongodb the first time it doesn't ask for the 0.0.0.0/0 IP, you can just select "Allow from anywhere". But after setup you can edit it under `Network Access`
I'll add a note right now in the Wiki.
![0000](https://user-images.githubusercontent.com/89225730/151488200-2968290e-f03a-42ea-a5b3-309e1eb8ac90.png)
